# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/epsilon/png-compile-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- epsilon-2004-01-18/src/lib/Makefile.am.orig	2004-01-19 16:42:53.000000000 +0100
+++ epsilon-2004-01-18/src/lib/Makefile.am	2004-01-19 16:43:06.000000000 +0100
@@ -5,7 +5,7 @@
 # A list of all the files in the current directory which can be regenerated
 MAINTAINERCLEANFILES = Makefile.in
 
-LDFLAGS		     = -L/usr/local/lib @imlib_libs@ @epeg_libs@
+LDFLAGS		     = -L/usr/local/lib @imlib_libs@ @epeg_libs@ -lz
 INCLUDES	     = -I/usr/local/include \
 		       -I$(includedir) @imlib_cflags@ @epeg_cflags@
 
